ispdb/umich.edu

```diff
@@ -1,22 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <clientConfig version="1.1">
-    <emailProvider id="umich.edu">
-        <domain>umich.edu</domain>
-        <displayName>UM ITCS Email</displayName>
-        <displayShortName>UM ITCS</displayShortName>
-        <incomingServer type="imap">
-            <hostname>mail.umich.edu</hostname>
-            <port>993</port>
-            <socketType>SSL</socketType>
-            <username>%EMAILLOCALPART%</username>
-            <authentication>password-cleartext</authentication>
-        </incomingServer>
-        <outgoingServer type="smtp">
-            <hostname>smtp.mail.umich.edu</hostname>
-            <port>587</port>
-            <socketType>STARTTLS</socketType>
-            <username>%EMAILLOCALPART%</username>
-            <authentication>password-cleartext</authentication>
-        </outgoingServer>
-    </emailProvider>
-</clientConfig>
\ No newline at end of file
+	<emailProvider id="umich.edu">
```

Comment:
Indention is wrong, 5 spaces. Please keep the indention, as it was, with 4 spaces.

Reply:
Thanks for the correction.